### PR TITLE
Create almost_done

### PR DESCRIPTION
**Pull request by wbathome**
Please review this, @casey-collab

### CODE REVIEW

File: almost_done [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (casey-collab) [inline almost_done]: Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.

Comment (casey-collab) [inline almost_done]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

Comment (casey-collab) [inline almost_done]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
This looks like a test pull request to me. That's fine, but I only want to merge contributions to the story, so I'm going to close it. If I've misunderstood, please re-open the pull request and let me know.

**Pull request closed without merging**